docs/articles/documentation/reference/configuration-file.md

```diff
@@ -1003,18 +1004,36 @@ Stops a test run if any test fails.
 
 ## tsConfigPath
 
-Enables TestCafe to use a custom [TypeScript configuration file](../guides/concepts/typescript-and-coffeescript.md#customize-compiler-options) and specifies its location.
+Deprecated as of [TestCafe version X.X.X.] in favour of the compilerOptions setting.
+
+*CLI*: [--ts-config-path](command-line-interface.md#--ts-config-path-path)  
+*API*: [runner.tsConfigPath](testcafe-api/runner/tsconfigpath.md)
+
+## compilerOptions
+
+Configures the [TypeScript compiler](../guides/concepts/typescript-and-coffeescript.md#customize-compiler-options).  The [official TypeScript documentation](https://www.typescriptlang.org/docs/handbook/compiler-options.html) contains the full list of configurable options.
```

Comment:
The `configPath` and `customCompilerModulePath` are TestCafe related options.
The `options` are options from [official TypeScript documentation](https://www.typescriptlang.org/docs/handbook/compiler-options.html).